whisper/whisperv5/peer_test.go

```diff
@@ -108,8 +108,7 @@ func TestSimulation(t *testing.T) {
 
 func initialize(t *testing.T) {
 	var err error
-	ip := net.IPv4(127, 0, 0, 1)
-	port0 := 30303
+	ip, port0 := getFreeAddress(t)
```

Comment:
`p2p.Server` sets `Port` to the actual port after startup. Please use `Port: 0` in config, then call `AddPeer` after they're all started.